src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -106,92 +106,92 @@
-                                        ;; have to do this by ID not ref, since the fields are generated within Table
-                                        (after-update #(.focus (.getElementById js/document "focus"))))}]])
-           [table/Table
-            {:key (str editing? (count (:attributes @state)))
-             :reorderable-columns? false :sortable-columns? (not editing?) :filterable? false :pagination :none
-             :empty-message "No Workspace Attributes defined"
-             :row-style {:alignItems "center" :fontSize "120%"}
-             :always-sort? (not editing?)
-             :header-row-style {:borderBottom (str "2px solid " (:line-default style/colors))
-                                :backgroundColor "white" :color "black" :fontWeight "bold"}
-             :resize-tab-color (:line-default style/colors)
-             :columns (if editing?
-                        [{:starting-width 40 :resizable? false :as-text (constantly "Delete")
-                          :content-renderer
-                          (fn [index]
-                            (icons/icon {:style {:color (:text-lightest style/colors)
-                                                 :verticalAlign "middle" :fontSize 22
-                                                 :cursor "pointer"}
-                                         :onClick #(swap! state update :attributes utils/delete index)}
-                                        :remove))}
-                         {:header (header "Key") :starting-width 300 :as-text (constantly nil)
-                          :content-renderer
-                          (fn [{:keys [key index]}]
-                            (style/create-text-field (merge
-                                                       {:style {:marginBottom 0 :fontSize "100%" :height 26 :width "calc(100% - 2px)"}
-                                                        :defaultValue key
-                                                        :onChange #(swap! state update-in [:attributes index]
-                                                                          assoc 0 (-> % .-target .-value))}
-                                                       (when (= index (-> (:attributes @state) count dec))
-                                                         {:id "focus"}))))}
-                         {:header (header "Value") :starting-width :remaining :as-text (constantly nil) :resizable? false
-                          :content-renderer
-                          (fn [{:keys [value index]}]
-                            (style/create-text-field {:style {:marginBottom 0 :fontSize "100%" :height 26 :width "calc(100% - 2px)"}
-                                                      :defaultValue value
+         [:div {}
+          (if editing?
+            [:div {:style {:marginBottom "0.25em"}}
+             [comps/Button {:icon :add-new :text "Add new"
```

Comment:
This is actually the only change in this file. If you want to reformat, please make a separate PR for that.